AddPathToPSModulePath.ps1

```diff
@@ -22,7 +22,7 @@ $paths = [Environment]::GetEnvironmentVariable("PSModulePath", "Machine").Split(
 if(!$paths.Contains($Path))
 {
     [System.Environment]::SetEnvironmentVariable("PSModulePath", [string]::Join(";", $paths + $Path), "Machine")
-    Write-Information "The path `"$Path`" was successfully added to the PSModulePath environment variable."
+    Write-Verbose "The path `"$Path`" was successfully added to the PSModulePath environment variable."
```

Comment:
Does `Write-Information` have a similar downside as `Write-Host`? I'd argue that in this case (and some other ones below) displaying a message is useful (so it should be `Write-Output`), even without `-Verbose` so that the user knows what's going on.

Personally I absolutely hate that printing nothing to the console means successful execution. It goes against UX and safety/security in so many other areas too.

Reply:
`Write-Information` outputs to a separate, object stream. It's rather for passing structured (and tagged) data. It's not problematic, like `Write-Host`, but it's not made for the purpose it's used here. `Write-Verbose` is the correct way to display status text. 

> Personally I absolutely hate that printing nothing to the console means successful execution.

Actually `$?` being `$True` is what means successful execution. :D It's extremely normal for scripts to not print anything on success, becuase you don't want to flood the log files with rubbish when they are executed in automation. Since scripts are meant to be automated, UX isn't really an issue and imo not leaking implementation information is a good security practice. I'm used to working with GNU utilities so this is the normal for me. [The dude who invented PowerShell agrees too. ](https://www.jsnover.com/blog/2013/12/07/write-host-considered-harmful/#:~:text=powershell%20is%20about%20automation.%20show%20comforting%20information%20is%20often%20helpful%20the%20first%20couple%20of%20times%20you%20run%20a%20script%20but%20then%20after%20that%2C%20it%20because%20an%20annoyance.)
You can always just call the cmdlet with `-Verbose` to retain the old behavior, and now all the status messages are nice and yellow so I think it's a win-win.

Reply:
That's a reasonable argument (I've read the article), but we should also consider that at least in Utility Scripts, many of these scripts are used to set up something by a user, not just for classic automation. But I agree with the sentiment about spamming the logs.

Please make a note in the Readme about the usage of the Verbose switch in case of an ambiguous or failed execution (i.e. that we conform to this convention about the usage of the Verbose switch).